draft-js-mention-plugin/src/MentionSuggestions/index.js

```diff
@@ -58,7 +58,9 @@ export default class MentionSuggestions extends Component {
           focusedOptionIndex: size - 1,
         });
       }
-
+      if (!this.props.store.getAllSearches().has(this.activeOffsetKey)) {
```

Comment:
it doesn't solve the multi-character mention trigger problem.  it just a simple protection for the error when componentDidUpdate try to get new getPortalClientRect but key already deleted by previous action. ( right now, it only can happened when set the mention trigger to be multi-characters. current mention plugin logic is kind of handle mention trigger as single char. for example:
`onSearchChange = (editorState, selection) => {
    const { word } = getSearchText(editorState, selection);
    const searchValue = word.substring(1, word.length);
    if (this.lastSearchValue !== searchValue) {
      this.lastSearchValue = searchValue;
      this.props.onSearchChange({ value: searchValue });
    }
  };`
`const selectionIsInsideWord = leaves
      .filter((leave) => leave !== undefined)
      .map(({ start, end }) => (
        start === 0 && anchorOffset === 1 && anchorOffset <= end || // @ is the first character
        anchorOffset > start + 1 && anchorOffset <= end // @ is in the text or at the end
      ))`

no sure if we need add support for mention trigger of multiple character